src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxIndexReadOperation.cs

```diff
@@ -1264,6 +1259,67 @@ public override IEnumerable<string> DynamicEntriesFields(HashSet<string> staticF
                 yield return field;
             }
         }
+        
+        internal struct GrowableHashSet<TItem>
```

Comment:
Why `struct`?

Reply:
Hmm, should be faster but since it can be problematic I reverted to class.